backend/buildtool/main.go

```diff
@@ -13,10 +13,10 @@ package main
 // cmd.Stdout = os.Stdout
 // cmd.Stderr = os.Stderr
 */
-import (
-	buildtool "github.com/benzend/goalboard/buildtool/runbuild"
-)
+// import (
+// 	buildtool "github.com/benzend/goalboard/buildtool/runbuild"
+// )
 
-func main() {
```

Comment:
No more buildtool?

Reply:
Build tool is still here as mentioned. Just focusing on core stuff atm with testing to move the project forward.